terraform/modules/quorum-cluster/variables.tf

```diff
@@ -460,3 +476,43 @@ variable "observer_node_counts" {
     sa-east-1      = 0
   }
 }
+
+# Backup Lambda related variables
+variable "backup_interval" {
+  description = "Schedule expression for backup"
+  default = ""
+}
+
+variable "backup_lambda_binary" {
+  default = ""
+  description = "Name of BackupLambda binary"
+}
+
+variable "backup_lambda_binary_url" {
+  default = ""
+  description = "Full URL path to Backup Lambda binary"
+}
+
+variable "backup_lambda_binary_path" {
```

Comment:
I'm searching through the PR and I can't find where this `backup_lambda_binary_path` variable is used. Please either point out where it is used, modify some hard-coded things to use this variable instead, or remove this variable completely.